docs/manual/src/proc_macro/index.md

```diff
@@ -131,6 +131,28 @@ fn do_something(foo: MyFooRef) {
 }
 ```
 
+### Default values
+
+Exported functions/methods can have default values using the `[uniffi(default = ...)]` attribute.
+
+```rust
+#[uniffi::export]
+fn split(
+    text: String,
+    #[uniffi(default = " ")]
+    sep=String,
+    #[uniffi(default = None)]
+    max_splits: Option<u32>,
+} {
+  ...
+}
+```
+
+Supported default values:
+  - String, integer, float, and boolean literals
+  - `[]` for empty Vecs
+  - `Option<T>` allows either `None` or a value for the inner type
```

Comment:
Without writing `Some()`? That seems strange to me.

Reply:
I was on the fence about that one.  I'll update it to use the `Some()` version.

Reply:
Thanks. I think it makes more sense that way because a builtin Rust feature for argument defaults would also almost definitely work that way.